dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/organisationunit/DefaultOrganisationUnitService.java

```diff
@@ -496,6 +499,42 @@ public boolean isInUserHierarchy( User user, OrganisationUnit organisationUnit )
         return organisationUnit.isDescendant( user.getOrganisationUnits() );
     }
 
+    @Override
+    @Transactional( readOnly = true )
+    public boolean isInUserViewHierarchy( OrganisationUnit organisationUnit )
```

Comment:
Can we rename to `isInUserDataViewHierarchy` here to be consistent with `User#dataViewOrganisationUnits` @abyot ?

Reply:
@larshelge, sure we can do.